types/logrotate-stream.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for logrotate-stream 0.2.5
```

Comment:
Why it is here instead of package.json `yarn add -D @types/logrotate-stream`? :)